test/models/item_test.rb

```diff
@@ -0,0 +1,13 @@
+require 'test_helper'
+
+class ItemTest < ActiveSupport::TestCase
+   test "create item with empty text and list reference" do
+    item = Item.new
+    assert item.invalid?
+    assert item.errors[:text].any?
+    assert item.errors[:list_id].any?
+    assert !item.save
+    assert_equal "can't be blank", item.errors[:text].join('; ')
+    assert_equal "can't be blank", item.errors[:list_id].join('; ')
```

Comment:
@vlasiak, Please consider the following suggestions:
- Let's split into two smaller tests ensuring that an item cannot be saved without `text` and `list_id`.
- Let's use a minimum number of asserts in order to test the behavior.
- use `factory_girl` gem instead of manual creation of models.